std/archive/tar.ts

```diff
@@ -53,28 +53,6 @@ export class FileReader implements Deno.Reader {
   }
 }
 
-/**
- * Simple file writer (call FileWriter.dispose() after use)
- */
-export class FileWriter implements Deno.Writer {
-  private file?: Deno.File;
-
-  constructor(private filePath: string, private mode: Deno.OpenMode = "w") {}
-
-  public async write(p: Uint8Array): Promise<number> {
-    if (!this.file) {
-      this.file = await Deno.open(this.filePath, this.mode);
-    }
-    return Deno.write(this.file.rid, p);
-  }
-
-  public dispose(): void {
-    if (!this.file) return;
-    Deno.close(this.file.rid);
-    this.file = undefined;
-  }
-}
```

Comment:
Unused? Why is it removed?

Reply:
Yes, unused and doesn't belong here if public.